fycon.go

```diff
@@ -0,0 +1,127 @@
+package desktop
+
+import (
+	"image/color"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/theme"
+	"fyne.io/fyne/widget"
+)
+
+type fyconRenderer struct {
+	objects []fyne.CanvasObject
+
+	image *Fycon
+}
+
+func (fy *fyconRenderer) MinSize() fyne.Size {
+	size := theme.IconInlineSize()
+	return fyne.NewSize(size, size)
+}
+
+func (fy *fyconRenderer) Layout(size fyne.Size) {
+	if len(fy.objects) == 0 {
+		return
+	}
+
+	fy.objects[0].Resize(size)
+}
+
+func (fy *fyconRenderer) Objects() []fyne.CanvasObject {
+	return fy.objects
+}
+
+func (fy *fyconRenderer) ApplyTheme() {
+	fy.Refresh()
+}
+
+func (fy *fyconRenderer) BackgroundColor() color.Color {
+	return color.Transparent
+}
+
+func (fy *fyconRenderer) Refresh() {
+	fy.objects = nil
+
+	if fy.image.Resource != nil {
+		raster := canvas.NewImageFromResource(fy.image.Resource)
+		raster.FillMode = canvas.ImageFillContain
+
+		fy.objects = append(fy.objects, raster)
+	}
+	fy.Layout(fy.image.Size())
+
+	canvas.Refresh(fy.image)
+}
+
+func (fy *fyconRenderer) Destroy() {
+}
+
+// Fycon widget is a basic image component that load's its resource to match the theme.
+type Fycon struct {
+	baseWidget
+
+	OnTapped      func()
+	Resource      fyne.Resource // The resource for this Fycon
```

Comment:
I don't think that this needs to be public - and the comment does not add anything here - something a little more meaningful - and include for the one above and below